hugo/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
-    "plotly.js": "1.51.3",
-    "parcel-bundler": "1.12.4"
+    "parcel-bundler": "1.12.4",
+    "plotly.js": "^2.5.1"
```

Comment:
Ich weiß nicht genau, inwiefern das dem selbst gebauten plotly in die Quere kommt, wozu ja auch ein Update-PR offen ist: s. `hugo/package.json` in https://github.com/codeformuenster/klimawatch/pull/132/files
Dort ist v.a. das Problem, dass die treemap nicht angezeigt wird. Das scheint ja kein Problem zu sein mit diesem hugo-update-PR. Vielleicht kann man dann diese Datei einfach nicht ändern?